packages/fxa-react/components/LogoLockup/index.tsx

```diff
@@ -0,0 +1,21 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React, { ReactElement } from 'react';
+import { ReactComponent as Logo } from '../../images/ff-logo.svg';
```

Comment:
Can this ff-logo not live adjacent to the component files? Ideally we are keeping component parts close to each other.

Reply:
We talked about doing that sometime ago in payments I believe and determined we wanted a separate `images` folder in case something needed to be reused. It could be argued that the logo could live in the `LogoLockup` directory, but we'd need to duplicate it if we needed it somewhere else, especially if for whatever reason we used a different component for the login flow or in payments down the line, and we want to reference just one SVG to make sure they're consistent... The `default-avatar` in particular is going to see a lot of reuse across the first `UnitRow` in the `Header`, and in the doorhanger drop down.

Reply:
Right, that makes sense, I just assumed we would default to standalone _until_ it needs to be re-used, instead of defaulting to shared. Because the opposite is also possible, where we think we might re-use something, leave it in shared images, and then never re-use it. Admittedly that's less likely to happen with the FF logo, but it gives me pause. Also, sorry if this sounds naggy 🙈

Reply:
It definitely doesn't sound naggy, this is definitely the time to bring up our conventions and file placements. :D I was mostly following our convention we set in payments - I personally think I'd prefer our images to be in one central spot than some in a central spot and some not, but if you prefer the latter, maybe make an issue so we can discuss / make that convention standard in payments as well?

Reply:
![image](https://user-images.githubusercontent.com/6392049/83047237-14635400-a016-11ea-8e46-1a5db6cd3ad4.png)